plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -39,6 +40,12 @@ class ServiceDestroyModal extends React.Component {
     });
   }
 
+  isGroupWithServices() {
+    const { service } = this.props;
+
+    return service instanceof ServiceTree && service.list.length > 0;
```

Comment:
Let's please add a `length` getter to the `Tree` resp. `List` struct instead of accessing the internals here.

Reply:
Added in the tree